src/components/FormAddRoom.jsx

```diff
@@ -0,0 +1,128 @@
+import React, { useState } from 'react';
+import { useCreateRoomMutation } from '../api/roomsData';
+
+const FormAddRoom = () => {
+  const [name, setName] = useState('');
+  const [description, setDescription] = useState('');
+  const [wifi, setWifi] = useState('');
+  const [tv, setTv] = useState('');
+  const [roomService, setRoomService] = useState('');
+  const [beds, setBeds] = useState(1);
+  const [imageUrl, setImageUrl] = useState('');
+  const [reserved, setReserved] = useState(false);
+
+  const [createRoom] = useCreateRoomMutation();
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+
+    createRoom({
+      name,
+      description,
+      wifi,
+      tv,
+      room_service: roomService,
+      beds,
+      image_url: imageUrl,
+      reserved,
+    });
+  };
```

Comment:
[OPTIONAL] Currently there is no way to tell if a room gets added. I think it would be a good idea to show a notification to your users or to redirect them to the main page after the successful addition of a room 👍